simtools/util/names.py

```diff
@@ -748,3 +755,44 @@ def camera_efficiency_log_file_name(site, telescope_model_name, zenith_angle, la
     name += f"_{label}" if label is not None else ""
     name += ".log"
     return name
+
+
+def get_telescope_type(telescope_name):
+    """
+    Guess telescope type from name. Types are "LST", "MST", "SST", "SCT".
+    Parameters
+    ----------
+    telescope_name: str
+        Telescope name
+    Returns
+    -------
+    str
+        Telescope type.
+    """
+
+    _class, _ = split_telescope_model_name(telescope_name)
+    try:
+        if _class[0:3] in ("LST", "MST", "SST", "SCT"):
```

Comment:
There is a names:all_telescope_class_names dict, should we use this here instead of listing all telescope types? If we hardcode the telescope types in gammasim-tools (which I don't like at all), we should have it at one place only.

Reply:
Good. I just adjusted to this dict